src/Tribe/Main.php

```diff
@@ -364,38 +369,80 @@ public function maybe_set_common_lib_info() {
 		}
 
 		/**
-		 * Plugins shouldn't include their functions before `plugins_loaded` because this will allow
-		 * better compatibility with the autoloader methods.
+		 * Resets the global common info back to ET's common path
 		 *
-		 * @return void
+		 * @since 4.9.3.2
 		 */
-		public function plugins_loaded() {
+		private function reset_common_lib_info_back_to_et() {
+			$et = Tribe__Tickets__Main::instance();
+			$et_common_version = file_get_contents( $et->plugin_path . 'common/src/Tribe/Main.php' );
```

Comment:
(1/2) apologies if I'm misreading but it looks like you're loading the Tribe__Main.php _source_ itself ...

Reply:
Ha! Well, at least we know that `version_compare()` identifies the entire source code as a greater version than that which is running. Great catch.